src/IB/FEMechanicsExplicitIntegrator.cpp

```diff
@@ -0,0 +1,647 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2014 - 2019 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+
+#include "ibamr/FEMechanicsBase.h"
+#include "ibamr/FEMechanicsExplicitIntegrator.h"
+#include "ibamr/ibamr_enums.h"
+#include "ibamr/ibamr_utilities.h"
+
+#include "ibtk/FEDataInterpolation.h"
+#include "ibtk/FEProjector.h"
+#include "ibtk/libmesh_utilities.h"
+
+#include "libmesh/boundary_info.h"
+#include "libmesh/enum_xdr_mode.h"
+#include "libmesh/equation_systems.h"
+#include "libmesh/quadrature_gauss.h"
+
+#include <IBTK_config.h>
+
```

Comment:
```suggestion
#include <IBTK_config.h>

```
These top-level includes don't exist in the CMake build system.

Reply:
Removed.